src/openboardview/main_opengl.cpp

```diff
@@ -360,9 +360,11 @@ int main(int argc, char **argv) {
 	cleanupAndExit(1);
 #endif
 
+	// SDL disables screen saver by default which doesn't make sense for us.
+	SDL_EnableScreenSaver();
+
 	ImGuiIO &io    = ImGui::GetIO();
 	io.IniFilename = NULL;
-	//	io.Fonts->AddFontDefault();
```

Comment:
Being nit-picky (as always!) but wouldn't this be better off in a separate cleanup commit? Probably ok to keep it in the PR though.